termwiz/src/caps/mod.rs

```diff
@@ -107,6 +107,8 @@ builder! {
 
         /// Whether mouse support is present and should be used
         mouse_reporting: Option<bool>,
+
+        force_terminfo_fallback_rendering: Option<bool>,
```

Comment:
Naming wise, how about this:

```suggestion
        /// When true, rather than using the terminfo `sgr` or `sgr0` entries,
        /// assume that the terminal is ANSI/ECMA-48 compliant for the
        /// common SGR attributes of bold, dim, reverse, underline, blink,
        /// invisible and reset, and directly emit those sequences.
        force_terminfo_render_to_use_ansi_sgr: Option<bool>,
```

Reply:
Added one extra sentence about why you might want to use it.